client/src/components/Add.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { useContext, useState } from "react";
+import { Box, Button, Paper, TextField, Stack } from "@mui/material";
+import { AppContext } from "../App";
+
+const Add = () => {
+  const [title, setTitle] = useState("");
+  const [url, setUrl] = useState("");
+  const { setVideos } = useContext(AppContext);
+
+  const addHandler = async (event) => {
+    event.preventDefault();
+    if (
+      title === "" ||
+      url === "" ||
+      title.trim() === "" ||
+      url.trim() === ""
+    ) {
+      alert("Please Fill All Fields!");
```

Comment:
Might be a nice touch to specify what fields need filling out; e.g. make an array of all the fields which don't satisfy the condition and display that to the user.